builtin/settingtypes.txt

```diff
@@ -800,6 +800,9 @@ serverlist_file (Serverlist file) string favoriteservers.txt
 #    Maximum size of the out chat queue. 0 to disable queueing and -1 to make the queue size unlimited
 max_out_chat_queue_size (Maximum size of the out chat queue) int 20
 
+#    Open the pause menu when the window's focus is lost. Does not pause if a formspec is open.
```

Comment:
Should be

> Does not pause if another menu/formspec is open

Reply:
for me this setting should be false by default, and shown in the main settings page, instead of advanced page

Reply:
I feel it's not important enough to be on the main settings page.